frontend/src/app/bounty/create/page.tsx

```diff
@@ -57,7 +54,17 @@ const CreateBountyPage: FC = () => {
     const [imgLink, setImgLink] = useState("");
     const [filename, setFilename] = useState<string | undefined>();
 
+    const [minDeadline, setMinDeadline] = useState<Date>(new Date());
+    const blockTimeStamp = useBlockTimestamp();
+    useEffect(() => {
+        if (blockTimeStamp !== undefined) {
+            const blockTimeStampInMS = new Date(Number(blockTimeStamp) * 1000);
+            blockTimeStampInMS.setDate(blockTimeStampInMS.getDate() + 1);
+            setMinDeadline(blockTimeStampInMS);
```

Comment:
I would rename `blockTimeStampInMS` as `blockDate`, since it's a `Date` object.

```suggestion
            const blockDate = new Date(Number(blockTimestamp) * 1000);
            blockDate.setDate(blockDate.getDate() + 1);
            setMinDeadline(blockDate);
```